mlserver/settings.py

```diff
@@ -44,6 +44,7 @@ class Config:
     uri: Optional[str] = None
     version: Optional[str] = None
     format: Optional[str] = None
+    extra: Optional[dict] = {}
```

Comment:
I think I missed this earlier. I think this is also another case of this https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

I'm not sure if pydantic allows this though?

Reply:
Yeah that's true. I will look into this

Reply:
This didn't seem to cause any issues so I'm going to assume this is okay

![image](https://user-images.githubusercontent.com/20640150/130634433-65f61d5a-c662-4430-9c9d-8e06f9b958b8.png)

Reply:
Would be to know if @adriangonz or @jklaise has better ideas on this.

Reply:
Yea, it appears that pydantic handles mutable defaults properly, alternatively one can use `default_factory`: https://stackoverflow.com/a/63808835